crates/nu_plugin_polars/src/dataframe/command/data/join.rs

```diff
@@ -172,6 +176,24 @@ impl PluginCommand for LazyJoin {
                     .into_value(Span::test_data()),
                 ),
             },
+            Example {
+                description: "Perform a full join of two dataframes and coalesce columns",
+                example: r#"let table1 = [[A B]; ["common" "common"] ["table1" "only"]] | polars into-df;
```

Comment:
can we remove the `;` from the end of the example lines? I don't think they're necessary and can be confusing to people. I noticed this the other day and changed a few.